demo-linux/demo.cpp

```diff
@@ -120,6 +144,8 @@ void demo()
             exit(-1);
         }
     }
+
+    writeToEEPROM();
```

Comment:
I think you're going to add some command-line args to this demo program anyway.  When you do, I might add a "--write-eeprom" flag to enable this.  Most people running a demo program probably don't expecting it to immediately destroy the contents of one of their EEPROM pages, which this will.

Reply:
This is still important to me.  We don't want a demo / test program to actually change the EEPROM of a connected spectrometer unless the user explicitly requests that.